File: android/src/main/java/com/majornutcracker/reactnativeselectablewebview/MajornutcrackerReactNativeSelectableTextModule.kt
package com.majornutcracker.reactnativeselectablewebview

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class MajornutcrackerReactNativeSelectableTextModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("MajornutcrackerReactNativeSelectableText")

    Constant("version") {
      "1.0.0"
    }

    Function("setMenuEnabled") { enabled: Boolean ->
      // Implementation would go here
    }
  }
}

File: ios/MajornutcrackerReactNativeSelectableTextModule.swift
import ExpoModulesCore
import WebKit

public class MajornutcrackerReactNativeSelectableTextModule: Module {
  public func definition() -> ModuleDefinition {
    Name("MajornutcrackerReactNativeSelectableText")

    Constant("version") {
      "1.0.0"
    }

    Function("setMenuEnabled") { (enabled: Bool) in
      WKWebView.isMenuEnabledGlobal(enabled)
    }
  }
}
